EnglishTranslation-master/ITEM.tsv

```diff
@@ -13214,17 +13214,17 @@ ITEM_20161214_013213	Revitalize your body and mind after a stressful exam period
 ITEM_20161214_013214	Straight A's Chocolate
 ITEM_20161214_013215	A sweet treat to stimulate brain activity. Increases movement speed by 2 for 30 minutes. Right-click to use.
 ITEM_20161214_013216	Token of the Demon Treaty
-ITEM_20161214_013217	A token made as a proof of the treaty made between the demons and Goddess.
+ITEM_20161214_013217	A token made as a proof of the treaty made between the demons and the goddesses.
 ITEM_20161214_013218	Token of the Goddess' Treaty
-ITEM_20161214_013219	A token made to signify the treaty between the demons and Goddess.
+ITEM_20161214_013219	A token made to signify the treaty between the demons and the goddesses.
 ITEM_20161214_013220	Treaty Slate
 ITEM_20161214_013221	A stone slate containing the terms of the treaty. Can only be read with the Tokens of the Treaty.
 ITEM_20161214_013222	Mihail's Review
 ITEM_20161214_013223	A review by Revelator Mihail, can be obtained as a part of a collection.
-ITEM_20161214_013224	The inscription on the ring.
-ITEM_20161214_013225	The note that has the inscription of the ring that a wandering soul on Letas Stream was obessessing over.  Can be added to your collection.
+ITEM_20161214_013224	Engraved Ring
```

Comment:
About this (also the description in ITEM_20161214_0132245 ) 
Its talking about the inscription of the ring. I don't think you need to change it to "engraved ring" since the item itself is the inscription of the ring, not the ring itself. (literally a paper note item with inscription of the ring)
You might want to recheck it.